www/src/components/layout.js

```diff
@@ -61,9 +62,9 @@ class DefaultLayout extends React.Component {
   }
 
   render() {
+    const itemList = getItemList(this.props.location.pathname)
     // SEE: template-docs-markdown for why this.props.isSidebarDisabled is here
```

Comment:
So... template-docs-markdown doesn't say anything about isSidebarDisabled and I don't see anywhere the prop is used. We might be able to delete this and get rid of the prop below.

Reply:
if a grep for this prop shows that it's not used anywhere I would agree to just remove it